operator/internal/manifests/internal/alerts/prometheus-alerts.yaml

```diff
@@ -209,3 +209,21 @@ groups:
     for: 1m
     labels:
       severity: warning
+  - alert: LokiIngesterCriticalFlushFailureRate
```

Comment:
Should we move it up before the "Lokistack" alert, so that it is grouped with the other `Loki*` alerts?